[CI:DOCS] UPDATE MANPAGE_SYNTAX (commit,attach,auto-update) (#10526)
containers/podman: docs/MANPAGE_SYNTAX.md

**podman subcommand command** [*optional*] *value* [*value* ...]

## DESCRIPTION

> **podman command** is always the beginning of the DESCRIPTION section. Putting the command as the first part of the DESCRIPTION ensures uniformity. All commands mentioned in a text retain their appearance and form. Commands that are quoted from other manpages have to be linked to those. None podman commands aren't to be linked but should be listed under SEE ALSO. This ensures clarity.\

Reviewer: I can't grok `None podman commands`. Is that supposed to be `Note`?

Reviewer: OH! I think you mean "Non-podman commands" then? Is that it?

Author: Yes I believe it is.

> **podman command** is always the beginning of the DESCRIPTION section. Putting the command as the first part of the DESCRIPTION ensures uniformity. All commands mentioned in a text retain their appearance and form. Commands that are quoted from other manpages have to be linked to those. None podman commands aren't to be linked but should be listed under SEE ALSO. This ensures clarity.\

Reviewer 2: ```suggestion
**podman command** is always the beginning of the DESCRIPTION section. Putting the command as the first part of the DESCRIPTION ensures uniformity. All commands mentioned in the text retain their appearance and form. Commands that are quoted from other manpages have to be linked to those manpages. Non-podman commands are not to be linked but should be listed under SEE ALSO. This ensures clarity.\
```

Example sentence: The command **podman command** is an example command.\

> It should also be specified if the command can only be run as root. In addition, it should be described when a command, OPTION, or other content cannot be executed with the remote client or in combination with other commands, OPTIONS, or contents. In this case the following sentence is put at the end of the first chapter of an command, OPTION, or content: *IMPORTANT: This option/command/other is not available with the command/OPTION/content/remote Podman client*. For a command, this should be done in the DESCRIPTION part. For the OPTIONS, it should be done in the DESCRIPTION of the specified OPTION. Do not use pronouns in the man pages, especially the word `you`.

Reviewer 2: ```suggestion
It should also be specified if the command can only be run as root. In addition, it should be described when a command, OPTION, or other content cannot be executed with the remote client or in combination with other commands, OPTIONS, or content. In this case, the following sentence is put at the end of the first chapter of a command, OPTION, or content: *IMPORTANT: This option/command/other is not available with the command/OPTION/content/remote Podman client*. For a command, this should be done in the DESCRIPTION section. For the OPTIONS, it should be done in the DESCRIPTION of the specified OPTION. Do not use pronouns in the man pages, especially the word `you`.
```

## OPTIONS

> All flags are referred to as OPTIONS. The term flags should not be used. All OPTIONS are listed in this section. OPTIONS that appear in descriptions of other OPTIONS and sections retain their appearance, for example: **--exit**. OPTIONS that are quoted from other manpages have to be linked to those with the command to execute them. OPTIONS within the same manpage don't have to be linked. This ensures clarity. Each OPTION should be explained to the fullest extend below the OPTION itself. Each OPTION is behind an H4-header (`####`). If the OPTION has a default argument, is has to be written in the description of the OPTION. If the OPTION is also not available with the remote client, the sentence about the default argument should the second to last sentence.

Reviewer 2: It would be good to put an example of a link for an option from another manpage.

> All flags are referred to as OPTIONS. The term flags should not be used. All OPTIONS are listed in this section. OPTIONS that appear in descriptions of other OPTIONS and sections retain their appearance, for example: **--exit**. OPTIONS that are quoted from other manpages have to be linked to those with the command to execute them. OPTIONS within the same manpage don't have to be linked. This ensures clarity. Each OPTION should be explained to the fullest extend below the OPTION itself. Each OPTION is behind an H4-header (`####`). If the OPTION has a default argument, is has to be written in the description of the OPTION. If the OPTION is also not available with the remote client, the sentence about the default argument should the second to last sentence.

Reviewer 2: ```suggestion
All flags are referred to as OPTIONS. The term flags should not be used. All OPTIONS are listed in this section. OPTIONS that appear in descriptions of other OPTIONS and sections retain their appearance, for example: **--exit**. OPTIONS that are quoted from other manpages have to be linked to those with the command to execute them. OPTIONS within the same manpage don't have to be linked. This ensures clarity. Each OPTION should be explained to the fullest extent below the OPTION itself. Each OPTION is behind an H4-header (`####`). If the OPTION has a default argument, it has to be written in the description of the OPTION. If the OPTION is also not available with the remote client, the sentence about the default argument should the second to last sentence.
```

#### **--version**, **-v** [=**false** | *true*]

> OPTIONS can be put after the command in two different ways. Eather the long version with **--option** or as the short version **-o**. If there are two ways to write an OPTION they are separated by a comma. If there are 2 (two) versions of one command the long version is always shown in front. If the OPTION is boolean, it is not requiered to set an argument. In this case the equal sign is put in brackets `[]` with the arguments, in difference to arguments that are not boolean as shown with **--answer**. The default argument that is shown for a boolean OPTION is given when the OPTION is not used in the command. If the OPTION is used in the command and no argument is specified, the OPTION will always become `true`.\

Reviewer: There is no reason for `2 (two)`, and it is awkward. Standard usage is to simply use the spelled-out number: "If there are two ..."

Reviewer: Typo: `requiered` should be `required`

Author: It was an idea. I wanted to see if this a good way to react on reading numbers wrong, but I guess you are right. In the end, it is just useless.

> OPTIONS can be put after the command in two different ways. Eather the long version with **--option** or as the short version **-o**. If there are two ways to write an OPTION they are separated by a comma. If there are 2 (two) versions of one command the long version is always shown in front. If the OPTION is boolean, it is not requiered to set an argument. In this case the equal sign is put in brackets `[]` with the arguments, in difference to arguments that are not boolean as shown with **--answer**. The default argument that is shown for a boolean OPTION is given when the OPTION is not used in the command. If the OPTION is used in the command and no argument is specified, the OPTION will always become `true`.\

Reviewer 2: ```suggestion
OPTIONS can be put after the command in two different ways. Eather the long version with **--option** or as the short version **-o**. If there are two ways to write an OPTION, they are separated by a comma. If there are two versions of one command the long version is always shown in front. If the OPTION is a boolean, it is not required to set an argument. In this case, the equal sign is put in brackets `[]` with the arguments.  This differs for arguments that are not boolean, as shown with **--answer** below. The default argument for a boolean OPTION is given when the OPTION is not used in the command. If the OPTION is used in the command and no argument is specified, the OPTION will always become `true`.\
```

Example: *IMPORTANT: This option is not available with the remote Podman client*

#### **--exit** [=**true** | *false*]

An example of an OPTION that has only one possible structure. Thus, it cannot be executed by the extension **-e**.

#### **--answer**=, **-a**=**active** | *disable*

> An example for an OPTIONS that accept 2 possible arguments as inputs is shown above. If there is a default argument that is selected when the OPTION is not used in the command, it is shown in **bold**. It must always be ensured that the standard argument is in the first place after the OPTION. In this example, there are 2 (two) different versions to execute the command. Both versions of the OPTION have to be shown with the arguments behind them. The default is **active**.

Reviewer: I can't parse this either. Maybe you mean "An example for an OPTION that accepts ..."?

Also, same comment about `2 (two)`.

Author: Grammar error on my side.

> An example for an OPTIONS that accept 2 possible arguments as inputs is shown above. If there is a default argument that is selected when the OPTION is not used in the command, it is shown in **bold**. It must always be ensured that the standard argument is in the first place after the OPTION. In this example, there are 2 (two) different versions to execute the command. Both versions of the OPTION have to be shown with the arguments behind them. The default is **active**.

Reviewer 2: ```suggestion
The "answer" option above is an example of an OPTION that accepts two possible arguments as inputs. If there is a default argument that is selected when the OPTION is not used in the command, it is shown in **bold**. It must always be ensured that the standard argument is in the first position after the OPTION. In this example, there are two different ways to execute the command. Both possible OPTIONs have to be shown with the arguments following them. The default is **active**.
```

Reviewer 2: I think this is what you were getting at.  ^^

#### **--status**=**good** | *better* | *best*

> This is an example for 3 (three) arguments behind an OPTION. If the number of arguments is higher than 3 (three), the arguments are **not** listed after the equal sign. The arguments have to be explained in a table like in **--test**=**_test_** regardless of the number of arguments. The default is **good**.

Reviewer: "greater than", not "higher than". And, same thing with the `3 (three)`.

> This is an example for 3 (three) arguments behind an OPTION. If the number of arguments is higher than 3 (three), the arguments are **not** listed after the equal sign. The arguments have to be explained in a table like in **--test**=**_test_** regardless of the number of arguments. The default is **good**.

Reviewer 2: ```suggestion
This is an example of three arguments following an OPTION. If the number of arguments is higher than three, the arguments are **not** listed after the equal sign. The arguments have to be shown in a table like the one below for **--test**=**_test_**, regardless of the number of arguments. The default value is shown as **good**.
```

#### **--test**=**test**

> OPTIONS that are followed by an equal sign include an argument after the equal sign in **bold**. If there is a default argument, that is used if the OPTION is not specified in the command, the argument after the eqaul sign is displayed in **bold**. All arguments must be listed and explained in the text below the OPTION. The default is **example one**.

Reviewer 2: ```suggestion
OPTIONS that are followed by an equal sign include an argument after the equal sign in **bold**. If there is a default argument, that is used if the OPTION is not specified in the command, the argument after the eqaul sign is displayed in **bold**. All arguments must be listed and explained in the text below the OPTION. The default value is shown as **example one**.
```

| Argument           | Description                                                                 |
| -                  | -                                                                           |

> | **example one**    | This argument is the standard argument if the OPTION is not specified.      |

Reviewer 2: standard argument or default argument.  I see it's pre-existing...   Hm...

| *example two*      | If one refers to a command, one should use **bold** marks.                  |
| *example three*    | Example: In combination with **podman command** highly effective.           |
| *example four*     | Example: Can be combined with **--exit**.                                   |
| *example five*     | The fifth description                                                       |

The table shows an example for a listing of arguments. The contents in the table should be aligned left. If the content in the table conflicts with this, it can be aligned in a way that supports the understanding of the content. If there is a standard argument, it **must** listed as the first entry in the table.

If the number of arguments is smaller than 4 (four) they have to be listed behind the OPTION as seen in the OPTION **--status**.

#### **--problem**=*container*

> OPTIONS that are followed by an equal sign that is followed by an unspecified argument, have no standard argument. If this OPTION is written with an equal sign and the argument is left empty, there will be no error, but the OPTION will be ignored. The meaning of the argument is described preferably in `one` word after the equal sign in *italic* format.

Reviewer 2: I'm a bit lost on this one.  I'm seeing `*container*` here, does this description pertain to `--problem`?
```suggestion
OPTIONS that are followed by an equal sign that is then followed by an unspecified argument, have no standard argument. If this OPTION is written with an equal sign and the argument is left empty, there will be no error, but the OPTION will be ignored. The meaning of the argument is described preferably in `one` word after the equal sign in *italic* format.
```
